packages/react-tabs/Spec.md

```diff
@@ -316,51 +344,61 @@ export type TabListProps = ComponentProps<TabListSlots> & {
+5. Subscribe to onTabSelected to show/hide content when a tab is selected.
 
 ### From v0
 
-- Replace use of `<Menu>` with `<TabList>` (full rewrite)
+1. Replace use of `<Menu>` with `<TabList>`
+2. Replace items data with Tab instances, writing a `map` function as needed.
 
 ## Behaviors
 
 ### Selection
 
-The TabList will provide single-selection of a tab.
+TabList provides single select of a tab.
 
-### Programmatic Selection
+Programmatic
 
-The TabList.selectedKey can control the currently selected tab.
-If the tab to be selected is in the overflow, it will be moved into the set of visible tabs.
+- The TabList.selectedKey can control the currently selected tab.
 
-> TODO: Do we need an announce here?
+Mouse/Touch
 
-### Overflow
+- Clicking the tab selects it.
 
-The overflow part will screen read as a button with text indicating to click for a list of other available tabs.
+Keyboard
 
-### Mouse/Touch
+- Tablist focus traps the and allows focus of individual tabs.
+- Arrow keys move focus forward/backward.
+- The arrow keys are based on layout. Horizontal: left/right, Vertical: up/down.
+- A tab is selected when the spacebar is pressed for a focused tab.
+- The `TAB` key moves focus from the tabs to the overflow button.
 
-Clicking the tab will raise the TabsList.onTabSelected event
+Events
 
-### Keyboard
+- Tab list raises the onTabSelected event whenever a tab is selected.
+- The event data includes the tab value.
 
-TabsList will focus trap on the set of tabs.
-When focused on the tabs, the user can cycle through tabs with arrow keys.
-The arrow keys are based on layout. Horizontal: left/right, Vertical: up/down
-The TAB key moves focus from tabs to overflow to next control in the tab order.
+### Overflow
 
-> TODO: Should Ctrl+Tab and Shift+Ctrl+Tab be supported?
+- When there are more tabs than can fit in the available space, the overflow appears.
+- Clicking the overflow button displays the non-visible tabs as a menu.
+- Selecting a tab from the menu will replace the last visible tab with the selected tab.
 
-### Screen readers
+## Accessibility
 
-Screen readers will read each tab's content when moused over similar to button.
-The ARIA tablist and tab roles will inform the screen reader.
+ARIA provides tab-specific roles:
 
-## Accessibility
+- [tab](https://www.w3.org/TR/wai-aria-1.1/#tab): A grouping label providing a mechanism for selecting the tab content that is to be rendered to the user.
+- [tablist](https://www.w3.org/TR/wai-aria-1.1/#tablist): A list of tab elements, which are references to tabpanel elements.
+- [tabpanel](https://www.w3.org/TR/wai-aria-1.1/#tabpanel): A container for the resources associated with a tab, where each tab is contained in a tablist.
 
 TabList and Tab will apply the `tablist` and `tab` ARIA roles.
-See the behaviors for selection, focus trap, and programmatic accessibility.
+The `tabpanel` will be reserved for possible future component.
+
```

Comment:
please add information about home/end keys and aria-selected state as mentioned in https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel